elyra/pipeline/processor_kfp.py

```diff
@@ -608,21 +608,34 @@ def _get_user_auth_session_cookie(url, username=None, password=None) -> dict:
         # try to authenticate using the provided credentials
         if 'dex/auth' in get_response.url:
             auth_info['endpoint_secured'] = True  # KF is secured
-
+            payload = {'login': username, 'password': password}
             # Try to authenticate user by sending a request to the
             # Dex redirect URL
             session = requests.Session()
             session.post(get_response.url,
-                         data={'login': username,
-                               'password': password})
+                         data=payload)
             # Capture authservice_session cookie, if one was returned
             # in the response
             cookie_auth_key = 'authservice_session'
             cookie_auth_value = session.cookies.get(cookie_auth_key)
 
+            # try dex local auth if existing method fails.
             if cookie_auth_value:
                 auth_info['authservice_session_cookie'] = \
                     f"{cookie_auth_key}={cookie_auth_value}"
+            else:
+                try:
+                    m = re.search('req=([^"]*)', get_response.text)
+                    req = m.group(0).replace("req=", "")
+                    r = session.post(f"{url}/dex/auth/local?req={req}", params=payload)
+                    if not r.ok:
+                        raise Exception(r.text)
```

Comment:
It would be better to not raise an exception (the calling method doesn't catch generic Exceptions) and instead utilize the existing error reporting approach https://github.com/elyra-ai/elyra/blob/12060dbf7b9273579da40ef9f15bf0d1e6efda14/elyra/pipeline/processor_kfp.py#L110-L111. One possible way to do that would be to extend the existing `auth-info` data structure with an optional property (e.g. `authservice_response`). The caller already logs the content of this data structure https://github.com/elyra-ai/elyra/blob/12060dbf7b9273579da40ef9f15bf0d1e6efda14/elyra/pipeline/processor_kfp.py#L115-L119